packages/core/stories/dialog/dialog.stories.tsx

```diff
@@ -18,6 +17,7 @@ import {
   useState,
 } from "react";
 import "./dialog.stories.css";
+import { CloseIcon } from "@salt-ds/icons";
```

Comment:
Is it worth to use icon from `useIcon` rather than import from icons package directly, to show the best practice?

Reply:
Likely storybook decorator doesn't include the semantic icon provider. this could be fine for now